src/app/pages/questions/components/question/timed-test/timed-test.component.ts

```diff
@@ -108,7 +108,7 @@ export class TimedTestComponent implements OnInit, OnChanges, OnDestroy {
       setTimeout(() => {
         this.timerTick()
       }, 1000)
```

Comment:
This piece of code needs to be removed now.

Reply:
Oops sorry I thought it was already removed. Fixed here: https://github.com/RADAR-base/RADAR-Questionnaire/pull/420/commits/f15cb32b02e1486021bd509644b2147d50e3e26d